mtp_cashbook/apps/cashbook/tests/test_functional.py

```diff
@@ -141,6 +141,7 @@ def test_help_popup(self):
     def test_go_back_home(self):
         self.driver.find_element_by_link_text('Home').click()
         self.assertEqual('Digital cashbook', self.driver.title)
+        self.assertEqual(self.driver.current_url, self.live_server_url + '/dashboard-batch-discard/')
```

Comment:
there's a helper for current URL assertions: `self.assertCurrentUrl('/dashboard-batch-discard/')`